RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -28,14 +28,20 @@ public struct CustomerCenterView: View {
 
     @StateObject private var viewModel: CustomerCenterViewModel
 
+    private var localization: CustomerCenterConfigData.Localization
+
     /// Create a view to handle common customer support tasks
-    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil) {
+    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil,
+                localization: CustomerCenterConfigData.Localization = .default) {
         self._viewModel = .init(wrappedValue:
                                     CustomerCenterViewModel(customerCenterActionHandler: customerCenterActionHandler))
+        self.localization = localization
     }
 
-    fileprivate init(viewModel: CustomerCenterViewModel) {
+    fileprivate init(viewModel: CustomerCenterViewModel,
+                     localization: CustomerCenterConfigData.Localization = .default) {
```

Comment:
I was wondering, I guess it's not possible to add a nullable value to the environment right? Feels it would be better than just defaulting to the default strings? As in, it feels we shouldn't have a default `Localization`...

Reply:
good point, wasn't happy about this either. Will check

Reply:
made it nullable!